collector/src/main/java/com/navercorp/pinpoint/collector/receiver/grpc/service/SimpleRequestHandlerAdaptor.java

```diff
@@ -49,7 +50,7 @@ public SimpleRequestHandlerAdaptor(String name, DispatchHandler<REQ, RES> dispat
     public void request(Message<? extends REQ> message, StreamObserver<? extends RES> responseObserver) {
         try {
             final ServerRequest<? extends REQ> request = serverRequestFactory.newServerRequest(message);
-            final ServerResponse<? extends RES> response = new GrpcServerResponse<>(responseObserver);
+            final ServerResponse<? extends RES> response = new GrpcServerResponse<>(responseObserver, ServerContext.getAgentInfo().isRetryFriendlyResponse());
```

Comment:
Could you check `create new GrpcServerResponse type for the RetryFriendlyResponse`?

Reply:
fixed with new GrpcRetryFriendlyServerResponse class